phpunit.xml.dist

```diff
@@ -9,11 +9,19 @@
          processIsolation="false"
          stopOnFailure="false"
          syntaxCheck="false"
-         bootstrap="tests/bootstrap.php"
+         bootstrap="vendor/symfony-cmf/testing/bootstrap/bootstrap.php"
 >
+    <php>
+        <server name="KERNEL_DIR" value="tests/Resources/app" />
+    </php>
+
     <testsuites>
-        <testsuite name="PHPCR Util Test Suite">
-            <directory>./tests/</directory>
+        <testsuite name="SonataDoctrinePHPCRAdminBundle Test Suite">
+            <directory>./tests/Unit</directory>
+        </testsuite>
+
+        <testsuite name="phpcr">
```

Comment:
No, sorry.

This is a kind of a failure in our `testing-bunlde`.  The name has to be `phpcr` or `orm` to start/init the database listener, to recognize the right one.